content/jsoc/gsoc/sciml.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: page
+title: SciML
+---
+
+# Science in Machine Learning 
+
+### Model Zoo on TPU
+
+Julia has experimental support for executing code on TPUs (https://github.com/JuliaTPU/XLA.jl) TPUs enable training cutting edge machine larning models written using Flux. However, TPUs are not able to execute arbitrary code and thus often require individual attention to fix new patterns in XLA.jl or other packages. Additionally, the performance characteristics of the TPU hardware are quite unlike that of CPU or even GPU and models may thus require TPU-specific adjustments to achieve peak performance. Lastly, the speed of TPUs presents signficant challenges to data input pipelines even at single-TPU levels of performance. Having a wide set of models available that are tuned for TPU will aid in finding common abstractions for models independent of hardware.
+
+Mentors: [Keno Fischer](https://github.com/Keno)
+
+### Benchmarks
+
+A benchmark suite would help us to keep Julia's performance for ML models in shape, as well as revealing opportunities for improvement. Like the model-zoo project, this would involve contributing standard models that exercise common ML use case (images, text etc) and profiling them. The project could extend to include improving performance where possible, or creating a "benchmarking CI" like Julia's own [nanosoldier](https://github.com/JuliaCI/Nanosoldier.jl).
+
+Mentors: [Dhairya Gandhi](https://github.com/dhairyagandhi96/), [Elliot Saba](https://github.com/staticfloat).
+
+### Multi-GPU training
```

Comment:
Do we not have a page just for all of the parallel stuff? That would probably make sense.

Reply:
We don't currently. Which ones fall under that category?

Reply:
Leave it here for now.